README.md

```diff
@@ -1,27 +1,4 @@
 <a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
```

Comment:
- [ ] Kindly make sure your linters are st correctly and no one fails to pass the test.
`Hint: You may consider reading the html-css linters documentation` [here](https://github.com/microverseinc/linters-config/tree/master/html-css#:~:text=README.md-,HTML%20%26%20CSS3%20Course,-If%20you%20are)